webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -533,7 +535,10 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
                 ct.go();
 
                 doc.newPage();
+
+                addScrambleSheet(doc, scrambleRequest, globalTitle, scramble, cb, i);
```

Comment:
I like that you created methods for these instead of adding them to this already huge function. Could you actually pull out the code that was already here into a method as well? If I understand the naming correctly, there are 3 things:

- Solution sheet with scramble (how about `addFmcSolutionSheetWithScramble()`?)
- Generic solution with without scramble (how about `addFmcSolutionSheetWithoutScramble()`?)
- Sheet with many copies of the same FMC scramble (how about `addFmcScrambleCutoutSheet()`?)

The names are kind of lengthy, but I think they're descriptive.